shared/common-adapters/meta.tsx

```diff
@@ -19,6 +19,7 @@ type Props = {
 const Meta = (props: Props) => (
   <Box2
     alignSelf="flex-start"
+    alignItems="center"
```

Comment:
did you check this change everywhere

Reply:
I checked it everywhere that uses a meta with an icon, and a few places that use a meta without an icon.